Containers/apache/Caddyfile

```diff
@@ -59,6 +59,13 @@
         reverse_proxy localhost:8000
     }
 
+    # Enable content encoding for javascript files
+    encode zstd gzip {
+        match {
+            header Content-Type application/javascript*
+        }
+    }
+
```

Comment:
@Zoey2936 unfortunately this does not work inside a route block where I would want to enable this specifically to keep the possible side-effects as low as possible. So I need to enable this globally unfortunately. 

Can you please check if this also works behind NPM or if we need to disable this when AIO is running behind a reverse proxy? Thank you!

Reply:
I've added `encode zstd gzip` above `# Collabora`, restartet the nextcloud-aio-apache container, checked if compression worked with `curl -L http://127.0.0.1:11000/login --header "Host: cloud.comain.de"` (it worked) and then tried using firefox to access the cloud behind my npm and yes it worked

Reply:
Nice! Thanks for verifying! Did you make sure to clean the browser cache after applying the setting?

Reply:
I've tried in Firefox "private" mode, I think a new private season has no cache, if I am right

Reply:
private mode has indeed no cache. Thanks for verifying! :)